wp-content/plugins/core/src/Routes/Sample_REST_Route.php

```diff
@@ -1,14 +1,11 @@
 <?php declare(strict_types=1);
 
-/**
```

Comment:
Why remove contextual comments?

Reply:
A comment that just repeats the class name that's right below it doesn't really provide any value and just adds noise to the code.